README.md

```diff
@@ -103,6 +103,7 @@ Conduct](https://pypeit.readthedocs.io/en/latest/codeconduct.html).
 * Gemini/GNIRS
 * Gemini/GMOS
 * Gemini/FLAMINGOS 2
+* GTC/OSIRIS
```

Comment:
Please also add `gtc_osiris` to the list under `docs/include/spectrographs_table.rst`

Reply:
Done!